### ISSUE TITLE
Missing Animal Spawns

### ISSUE THREAD

**Issue by cheesykyle** [TayMcKenzieNZ/server_scenarios#3]
Have you experienced issues with missing animal spawns across the map when streaming modified scenarios? I believe this is a common issue and perhaps it's just all unmodified scenarios aren't properly streaming but I thought this was supposed to alleviate not needing to stream the default scenarios.

Appreciate the research and effort you've put into this subject!

**Comment by cheesykyle** [comment]
I see the error of my ways. It should probably be more clear in the documentation, but you still need to define any/all of the scenarios you want to use in the sp_manifest even if you're not streaming them, including vanilla/default ones.

**Comment by TayMcKenzieNZ** [comment]
Yeah, I've tried this "ou still need to define any/all of the scenarios you want to use in the sp_manifest even if you're not streaming them, including vanilla/default ones" however still had issues.

To be honest, unfortunately there isn't all too much research done surrounding scenario streaming to fivem, and I have noticed gangs disappeared also.

The mod creators for the Rags And Riches Mile High Club have a custom scenario resource but are too snobby and overprotective with leaks, that they have put their crap behind pay walls and encryption, and obviously don't believe in open source software.

I have unfortunately at this stage given up on this resource and allow people to fork it and figure out what the heck to do, because I am all out of ideas.
